package.json

```diff
@@ -35,60 +35,63 @@
     "react": "15.5.4",
     "react-dom": "15.5.4",
     "react-redux": "5.0.5",
-    "react-router": "3.0.5",
-    "react-router-redux": "4.0.8",
+    "react-router": "4.2.0",
+    "react-router-dom": "4.0.0",
+    "react-router-redux": "5.0.0-alpha.6",
```

Comment:
Just want to ask this again before we move forward...

Are we sure we want to keep this?

Reply:
cc: @coryhouse

Reply:
I vote for dropping it. Easy to add if desired, and avoids an alpha dep